src/components/GenreSelect/GenreSelect.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { GenreSelect } from './GenreSelect';
+
+describe('GenreSelect Component', () => {
+  const genres = ['Action', 'Drama', 'Comedy'];
+  const currentGenre = 'Action';
+  const selectedGenreCallback = jest.fn();
+
+  it('renders all genres passed in props', () => {
+    const { getAllByText } = render(
+        <GenreSelect genres={genres} current={currentGenre} selectedGenreCallback={selectedGenreCallback} />
+    );
+
+    const genreItems = getAllByText((text) => genres.includes(text));
+    expect(genreItems.length).toBe(genres.length);
```

Comment:
nit: You are checking length, is not "strictly" the same value. You could check length and the arrays.

Reply:
> Test that component renders all genres passed in props